capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js

```diff
@@ -174,13 +174,58 @@ function createNewsFeed(articles) {
   return newsFeed;
 }
 
+/** Converts post objects into links to be featured under the "Posts" tab.
+ * @param {Array.<Object>} schoolPosts - A list of posts pertaining to a school.
+ * @param {string} schoolName - The name of a school.
+ * @return {string} - The HTML code for providing either post links or an option
+ * to make the first post for a school.
+ */
+function createLinks(schoolPosts, schoolName) {
+  // If there are no existing posts for a school, provide the user with a link
+  // to the page where they can create one.
+  if (schoolPosts === undefined|| schoolPosts.length == 0) {
```

Comment:
nit: space between undefined and || also strongly type check the length with triple equals `===`